dashboard/config/locales/en.yml

```
@@ -1402,3 +1402,10 @@ en:
     thank_you: "THANK YOU for championing computer science education. Our team wishes you and your students an unforgettable Hour of Code!"
     founder: "Founder, Code.org"
     your_friends: "Your Friends at Code.org"
+  lti:
+    error:
+      missing_params: "Missing required param(s): client_id, lms, email"
+      unsupported_lms_type: "Unsupported LMS platform type"
+      integration_exists: "Lti Integration already exists for this Client ID"
+      wrong_resource_type: "Only LtiResourceLink is supported right now"
+    create_integration_success: "Succesfully created your LTI integration"
```

This looks great, the only nit would be to keep strings alphabetically ordered.
```suggestion
  lti:
    create_integration_success: "Successfully created your LTI integration"
    error:
      integration_exists: "Lti Integration already exists for this Client ID"
      missing_params: "Missing required param(s): client_id, lms, email"
      unsupported_lms_type: "Unsupported LMS platform type"
      wrong_resource_type: "Only LtiResourceLink is supported right now"
```

---

@mgc1194 is that because we alphabetize keys somewhere in the sync output files, and we're trying to minimize the amount of processing we have to do?

---

Generally, we sort the strings that are synced, but in this case, the original file is not modified. For discoverability is better to have them alphabetically, otherwise is more likely to end up having duplicated strings.